### PR TITLE
[INCPP122DG-267] Fix Schema resolution

### PR DESCRIPTION
**Pull request by FlorentinPG**
## Description
Se modifica la librería que se utiliza para validar los JSON Schema de las credenciales con objeto de resolver un problema de resolución interna de referencias y también la obtención de esquemas remotos.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [X] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
[INCPP122DG-267](https://wealize.atlassian.net/browse/INCPP122DG-267)

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs in project
- [ ] 🍕 Shared with team / daily
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?

  

[INCPP122DG-267]: https://wealize.atlassian.net/browse/INCPP122DG-267?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json
Comment (yerayrodriguez) [inline package.json:1]: No has eliminado la dependencia antigua a jsonschema

File: src/core/presentations/vp-resolver.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
   Schema,
   Validator
 } from "jsonschema";
```
Comment (yerayrodriguez) [inline src/core/presentations/vp-resolver.ts]: No has eliminado la dependencia con jsonschema

Comment (yerayrodriguez) [reply src/core/presentations/vp-resolver.ts]: Mirando el código de este fichero, simplemente usas Schema para una conversión de tipo

**Review by yerayrodriguez — CHANGES_REQUESTED** [review]

File: package.json
Comment (FlorentinPG) [reply package.json:1]: Hecho

File: src/core/presentations/vp-resolver.ts (hunk 1 above)
Comment (FlorentinPG) [reply src/core/presentations/vp-resolver.ts]: Quitado Schema

**Review by yerayrodriguez — APPROVED** [review]

**Pull request merged**